res/skins/QMLDemo/Mixxx/Controls/Knob.qml

```diff
@@ -70,29 +60,32 @@ Item {
 
     }
 
-    MouseArea {
-        id: mousearea
-
-        property real posy: root.height / 2
+    DragHandler {
```

Comment:
The mouse is for the wheel handling (WheelHandler does not exist in Qt 5.12).

The DragHandler is for modifying the knob value when it's turned (cpress + move = drag).

Using only MouseAreas doesn't seem to be possible, because there doesn't seem to be away to pass through simple events to underlying MouseAreas (e. g. a mouse area that reacts to doubleclicks in ResettableControlKnob breaks the mousearea responsible to turning in the  Knob component, because the click events are not propagated).

Reply:
Not sure if it would solve the problem but have you read the docs on [MouseArea.propagateComposedEvents](https://doc.qt.io/qt-5/qml-qtquick-mousearea.html#propagateComposedEvents-prop) and [MouseEvent.accepted](https://doc.qt.io/qt-5/qml-qtquick-mouseevent.html#accepted-prop)?

Reply:
I did and spent quite some time on debugging. The first doesn't work because it only works for *composed* events, not simple events. I can pass though a double click to from an upper mousearea to a lower one, but not a simple press because it's not composed.

If I set mouseevent.accepted = false in the onPressed handler of the upper mousearea, the event is passed through, but then the upper mousearea doesn't react to double clicks anymore, because the first press was already not accepted.

Reply:
Ok, then the current solution is fine, though it would be nice if you could document that the mousearea is just used as a replacement WheelHandler until 5.15 is the minimum version.